src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingOverrideCheck.java

```diff
@@ -72,6 +72,32 @@
  * <pre>
  * &lt;module name=&quot;MissingOverride&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class Test extends SuperClass {
+ *
+ *     &#47;** {&#64;inheritDoc} *&#47;
+ *     &#64;Override
+ *     public void test1() { // OK
+ *
+ *     }
+ *
+ *     &#47;** {&#64;inheritDoc} *&#47;
+ *     public void test2() { // violation, should be annotated with &#64;Override
+ *
+ *     }
+ *
+ *     &#47;** {&#64;inheritDoc} *&#47;
+ *     private void test3() { // violation, using the &#64;inheritDoc tag on private method
+ *                            // that is not valid
+ *     }
+ *
+ *     &#47;** {&#64;inheritDoc} *&#47;
+ *     public static void test4() { // violation, using the &#64;inheritDoc tag on static method
+ *                                  // that is not valid
```

Comment:
`// that is not valid` I think this can be removed

Reply:
done